app/views_model.py

```diff
@@ -30,24 +30,6 @@ def get_cities(select="*", artist_id=None):
     result = models.engine.execute(sql)
     return result
 
-
-#query not in use yet
```

Comment:
It's always an extremely good idea to follow the `YAGNI` principle: You Ain't Gonna Need It. Use only as many functions as you need; the ones you write before they are needed will, probably, never be needed, or when they are needed they won't be in the form you predicted.